vortex-array/src/stats/stats_set.rs

```diff
@@ -47,38 +47,6 @@ impl StatsSet {
         StatsSet::new_unchecked(vec![(Stat::NullCount, Precision::exact(0))])
     }
 
-    pub fn constant(scalar: Scalar, length: usize) -> Self {
```

Comment:
was this unused?cccccbngjuecgil

Reply:
This was only ever used in ConstantArray::new which no longer by default populates stats and instead relies on compute functions